app/controllers/application_controller.rb

```diff
@@ -1,5 +1,13 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    User.first
-  end
+  before_action :authenticate_user!
+  # protect_from_forgery with: :exception
+
+  # before_action :update_allowed_parameters, if: :devise_controller?
+
+  # protected
+
+  # def update_allowed_parameters
+  #   devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:name, :photo, :bio, :email, :password) }
+  #   devise_parameter_sanitizer.permit(:account_update) { |u| u.permit(:name, :photo, :bio, :email, :password, :current_password) }
+  # end
```

Comment:
- [ ] this shouldn't be commented especially the update_allowed_parameters which allow you to add input in registrations like name bio etc .... so please try to uncomment it to fix it

Reply:
alright, let me do that